src/mock/index.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
+const fs = require('fs');
+const axios = require('axios'); // eslint-disable-line
+const dataDir = require('data-dir'); // eslint-disable-line
+
+const { emulators } = require('../../firebase.json');
+const { region } = require('../../functions/config');
+
+const { projects } = JSON.parse(fs.readFileSync(path.join(__dirname, '../../.firebaserc'), 'utf8'));
+
+// Emulator settings for functions
+const PORT = emulators.functions.port;
+const PROJECT_ID = projects.development;
+const REGION = region;
+const FUNCTION_ID = 'populateFirestoreEmulator';
+const SECRET = 'UZPmJ9gOXHmb6RRttAyURi4JdkvDq8'; // todo: read from environment variable
+const url = `http://localhost:${PORT}/${PROJECT_ID}/${REGION}/${FUNCTION_ID}?secret=${SECRET}`;
+
+run();
+
+async function run() {
+  let users;
+
+  try {
+    const systemUsers = require('./users/systemUsers.json');
+    const customUsers = require('./users/customUsers.json');
```

Comment:
the actual file is named customUsers-template.json and not customUsers.json

Reply:
requires the file inside a Promise() now which fixes this issue